package.json

```diff
@@ -5,7 +5,7 @@
   "main": "build/cp-multi-selector.js",
   "scripts": {
     "test": "node node_modules/karma/bin/karma start",
-    "build": "webpack --optimize-minimize"
+    "build": "webpack"
```

Comment:
It's 24kb uncompressed, I figured not a big deal to not compress it. Would have saved me time debugging if it was not compressed when I was finding the problem